Arena.toml

```diff
@@ -1067,6 +1067,10 @@ message = "star.wdl:61:13: warning[SnakeCase]: input name `genome_SA_index_n_bas
 document = "stjudecloud/workflows:/tools/star.wdl"
 message = "star.wdl:62:13: warning[SnakeCase]: input name `genome_SA_sparse_d` is not snake_case"
 
+[[diagnostics]]
+document = "stjudecloud/workflows:/tools/star.wdl"
+message = "star.wdl:816:8: warning[PascalCase]: struct name `SJMotifs` is not PascalCase"
```

Comment:
I'm going to hop over to `workflows` and fix this, so that this diagnostic doesn't show up on `main`. Will finish my review after that, but here's a partial reveiw.